samples/weather_function/src/main/kotlin/org/example/weather_func/curl.kt

```diff
@@ -0,0 +1,89 @@
+package org.example.weather_func
+
+import org.example.weather_func.Event
+// size_t is a alias for Long.
+import platform.posix.size_t
+
+import curl.curl_easy_setopt as setCurlOption
```

Comment:
Not sure if `import smth as smth_else` is generally a good practise, as introduces uneeded level of indirection.